### ISSUE TITLE
Confused

### ISSUE THREAD

**Issue by alphapapa** [jtkDvlp/simple-bookmarks#4]
Hi, this package looks interesting, but I don't understand what it offers above packages like Bookmarks+ and helm-bookmarks.  It also looks a bit overwhelming with all those binds in the readme.  Could you provide some examples of what it can do that those other packages can't?

**Comment by jtkDvlp** [comment]
> Hi, this package looks interesting, but I don't understand what it offers above packages like Bookmarks+ and helm-bookmarks. ... Could you provide some examples of what it can do that those other packages can't?

I think there is no direct offer above existing packages like bookmark+ nor helm-bookmark by my package. Presumably vice versa, these packages offer more like listing, filtering, editing, sorting etc. And **this** ist the point. My package offers simplicity! The functionality and its source code is simple. As explained in the readme: "It provides a simple but convenient interface to your recently and often used bookmarks / functioncalls." Means: no much code, no extravagant menus nor abilities to sort or edit etc. Handling / getting the bookmark I want to, is done by the mini-buffer completion package not by my package (in my case ivy) :-)

So why I decided to write this package:
- make my first steps into Elisp
- get a simple and convenient tool for bookmarks **and** any functioncalls
- get a tool that can handle desktops / sessions like a bookmark (previously I used bookmark+ but it didn´t work as expected for me)

> It also looks a bit overwhelming with all those binds in the readme. 

That´s true. It´s a big block of bindings. You don´t need them all. The first five bindings would be enough. The others are just for prefiltering unless wished.

Maybe I set an minor mode up on it, so users just need to turn the mode on to use the default bindings.

Are your questions answered so?
